modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationTemplateLocalServiceImpl.java

```diff
@@ -98,10 +102,10 @@ public NotificationTemplate addNotificationTemplate(
 		for (NotificationRecipientSetting notificationRecipientSetting :
 				notificationContext.getNotificationRecipientSettings()) {
 
-			notificationRecipientSetting.setNotificationRecipientId(
```

Comment:
Hey @murilostodolni , did you change this order intentionally?

Reply:
Hi @guilhermedcamacho the changed was made by `source-format `